### PR TITLE
8021qaz: Add parameter passive_8021qaz to run in read only mode

### PR DESCRIPTION
**Pull request by imanti**
Add parameter passive_8021qaz to change 8021qaz module behavior
with respect to hw values. If passive_8021qaz is true, then hw
values will be updated/modified. If passive_dcbx is false, then
there will be no changes to hw.

To run in passive mode use -P option: 'lldpad -P'

### CODE REVIEW

**Comment by dmertman** [comment]
Sorry for delayed response.  There is an issue with this pull-request.  With the -P parameter used, if a lldpad command is issued to change the ETS configuration:  The application of the new values is blocked, but the advertised values are updated to the requested values.

**Comment by imanti** [comment]
@dmertman, added condition to block calls to 8021qaz set functions if passive mode is enabled.

**Comment by imanti** [comment]
@paravmellanox, Parameter name change from 'passive_8021qaz' to 'read_only_8021qaz'
@dmertman, The changes you requested are in the last commit

File: lldpad.c [hunk 1]
```diff
@@ -262,7 +262,7 @@ int main(int argc, char *argv[])
 			pid_file = 0;
 			break;
 		case 'P':
```
Comment (paravmellanox) [inline lldpad.c]: @imanti please change this to 'R'

Comment (imanti) [reply lldpad.c]: Done!

Comment (paravmellanox) [reply lldpad.c]: @imanti did you push the change? I don't see here.

Comment (imanti) [reply lldpad.c]: I did. I don't know why if you go directly to the last commit the changes are not updated. If you check the PR files changed tab you can see the change.

**Comment by praeluceo** [comment]
We'll give this a test tomorrow!

**Comment by imanti** [comment]
@praeluceo, @paravmellanox. I've updated commit messages

**Comment by praeluceo** [comment]
We have confirmed that with the lldpad -R flag set, commands that change the tlv fields are blocked; which should prevent advertising values which aren't actually applied:
{{[root@LIN-ND2-075 openlldp]# cat ./apply_3tc.sh }}
./lldptool -Ti enp129s0f0 -V ETS-CFG enableTx=yes willing=no up2tc=0:0,1:0,2:0,3:1,4:0,5:0,6:0,7:0 tcbw=25,75 tsa=0:ets,1:ets
./lldptool -Ti enp129s0f0 -V ETS-REC enableTx=yes up2tc=0:0,1:0,2:0,3:1,4:0,5:0,6:0,7:0 tcbw=25,75 tsa=0:ets,1:ets
./lldptool -Ti enp129s0f0 -V PFC enable=yes enableTx=yes willing=no enabled=3#lldptool -Ti enp129s0f0 -V ETS-CFG enableTx=yes willing=yes
#lldptool -Ti enp129s0f0 -V PFC enable=yes enableTx=yes willing=yes
{{[root@LIN-ND2-075 openlldp]# ./apply_3tc.sh }}
Access denied 80c209
Access denied 80c20a
Access denied 80c20b
 
Looks fine to us.

**Pull request merged**